RELEASE.md

```diff
@@ -277,17 +286,26 @@ After the release completes:
 
 - Update the
   [community-operator](https://github.com/k8s-operatorhub/community-operators)
-  and
+    and
```

Comment:
nit: There is some weird spacing in this bullet. Doesn't effect the rendering, but I think the 2 spaces added on this line should be undone and next two lines should each have two spaces deleted.

Reply:
Updated. This was done by my "Rewrap" plugin.  I guess it got confused by the long links.